test/tokend-transformer.js

```diff
@@ -139,7 +139,8 @@ describe('TokendTransformer', function() {
         $tokend: {
           type: 'kms',
           resource: '/v1/kms/decrypt',
-          ciphertext: 'gbbe'
+          ciphertext: 'gbbe',
+          region: 'eu-central-1'
```

Comment:
I'd argue that another test case doesn't necessarily illustrate anything if we're not requiring the region with the request. Remember that [Tokend sets a default value](https://github.com/rapid7/tokend/blob/master/lib/providers/kms.js#L21-L28) if there's no `region` in the request.